packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -447,20 +486,25 @@ class C4DMastheadL1 extends StableSelectorMixin(LitElement) {
 
     return html`
       <li
+        part="masthead-l1-item"
         @focusout=${handleDropdownClose}
         @keydown=${handleDropdownClose}
         tabindex="-1">
         <button
+          part="masthead-l1-item-button"
           class="${prefix}--masthead__l1-item"
           @click=${toggleSubsection}
           @focusin=${handleTopNavFocusIn}>
           ${title}${ChevronDown16()}
         </button>
         <div
+          part="l1-dropdown"
           data-dropdown-target
           class="${prefix}--masthead__l1-dropdown ${prefix}--masthead__l1-dropdown__${columns}-col">
           ${announcement
-            ? html`<div class="${prefix}--masthead__l1-dropdown-announcement">
+            ? html`<div
+                part="l1-dropdown-announcement"
+                class="${prefix}--masthead__l1-dropdown-announcement">
```

Comment:
Not able to comment directly on it, but line 511 below is missing a `part` attribute. Probably `dropdown-links` would work.